app/Http/Controllers/UserController.php

```diff
@@ -125,6 +125,7 @@ public function update(Request $request, $id)
         $validator = Validator::make($request->all(), [
             'name' => 'required|string|max:255',
             'email' => 'required|string|email|max:255|unique:users,email,'.$user->id,
+            'password' => 'string|confirmed',
```

Comment:
Can you use the same validation for password that is used for registering. Here is what it is from the auth controller: required|string|min:6|confirmed

Reply:
Also when the password and confirm password are empty I get an error saying the password is not a string. You might need to add to the password validator nullable, or sometimes, or create a sometimes method (https://laravel.com/docs/5.5/validation#conditionally-adding-rules).

Reply:
fixed in 04ed2ca, thanks for the conditional validation tip!